includes/Modules/Analytics/Advanced_Tracking/Plugin_Detector.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Plugin_Detector
+ *
+ * @package   Google\Site_Kit\Modules\Analytics
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+// phpcs:disable WordPressVIPMinimum.Constants.ConstantString.NotCheckingConstantName
+// phpcs:disable WordPress.NamingConventions.ValidVariableName.PropertyNotSnakeCase
+// phpcs:disable WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
```

Comment:
PHPCS rules shouldn't be disabled. If a specific violation is flagged which is actually a false positive, let's use `phpcs:ignore` on the individual line.

Reply:
`phpcs: ignore` seems only to ignore the warnings. In the plugin detector, however, it is an error that we are trying to ignore, which `phpcs:ignore` doesn't seem to work in this case. The error is: Constant name, as a string, should be used along with `defined()`.

Reply:
`phpcs:ignore` should ignore both errors and warnings, can you double-check where you put it? It needs to be on the same or right above the line where the error is flagged.